projects/kit/src/lib/constants/default-decimal-pseudo-separators.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_PSEUDO_SEPARATORS = ['.', ',', 'б', 'ю'];
```

Comment:
It is not obvious what kind these separators are.
Let's change in the following way:

```suggestion
export const DEFAULT_DECIMAL_PSEUDO_SEPARATORS = ['.', ',', 'б', 'ю'];
```